users/models.py

```diff
@@ -225,20 +225,24 @@ class FaultyOpenEdxUserManager(BaseUserManager):
     """User manager that defines a queryset of Users that are incorrectly configured in the openedx"""
 
     def get_queryset(self):  # pylint: disable=missing-docstring
-        return (
+        faulty_users = (
             super()
             .get_queryset()
             .select_related("openedx_api_auth")
-            .prefetch_related("openedx_users")
+            .prefetch_related("openedx_users", "openedx_users__has_been_synced")
```

Comment:
This new prefetch isn't needed since the record is already prefetched with the first arg.